gcloud/monitoring/timeseries.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Time series for the `Google Monitoring API (V3)`_.
+
+.. _Google Monitoring API (V3):
+    https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+"""
+
+# Features intentionally omitted from this first version of the client library:
+#   - Creating time series.
+#   - Natural representation of distribution values.
```

Comment:
Why not put this in the docstring? Is there an issue tracker where there is a tracking bug for these omissions?